python/ray/rllib/ppo/filter.py

```diff
@@ -92,16 +92,21 @@ def __init__(self, shape, demean=True, destd=True, clip=10.0):
         self.destd = destd
         self.clip = clip
         self.rs = RunningStat(shape)
+        self.buffer = RunningStat(shape)
+
+    def clear_buffer(self):
```

Comment:
Is this called anywhere?

Reply:
I see, we don't need it for PPO because we always just create a new RunningStat object for each rollout, right?